makefile

```diff
@@ -1,4 +1,4 @@
-MORELLO_SDK?=${HOME}/cheri/output/morello-sdk
+MORELLO_SDK?=${HOME}/cheri/output-cornucopia/morello-sdk
```

Comment:
I would recommend leaving this default as it is the default sdk path. It can always be overriden by an env